notes/returned-mail-received.vbs

```diff
@@ -55,26 +56,36 @@ changelog_display
 'END CHANGELOG BLOCK =======================================================================================================
 
 'THE SCRIPT-----------------------------------------------------------------------------------------------------------------
-'Connects to BLUEZONE
-EMConnect ""
-'Grabs the MAXIS case number
-CALL MAXIS_case_number_finder(MAXIS_case_number)
+
+EMConnect ""                                        'Connecting to BlueZone
+CALL MAXIS_case_number_finder(MAXIS_case_number)    'Grabbing the CASE Number
+call Check_for_MAXIS(false)                         'Ensuring we are not passworded out
+back_to_self                                        'added to ensure we have the time to update and send the case in the background
 
 '-------------------------------------------------------------------------------------------------DIALOG
-Dialog1 = "" 'Blanking out previous dialog detail
-'Running the initial dialog
-BeginDialog Dialog1, 0, 0, 211, 85, "Returned Mail"
-  EditBox 55, 5, 40, 15, MAXIS_case_number
-  EditBox 155, 5, 50, 15, date_received
-  DropListBox 90, 25, 115, 15, "Select One:"+chr(9)+"forwarding address in MN"+chr(9)+"forwarding address outside MN"+chr(9)+"no forwarding address provided"+chr(9)+"no response received", ADDR_actions
-  EditBox 90, 45, 115, 15, worker_signature
+Dialog1 = "" 'Running the initial dialog
+BeginDialog Dialog1, 0, 0, 221, 215, "RETURNED MAIL PROCESSING"
+  EditBox 60, 5, 50, 15, MAXIS_case_number
+  EditBox 60, 25, 50, 15, date_received
+  DropListBox 105, 65, 110, 15, "Select:"+chr(9)+"forwarding address in MN"+chr(9)+"forwarding address outside MN"+chr(9)+"no forwarding address provided"+chr(9)+"no response received", ADDR_actions
+  DropListBox 100, 150, 110, 15, "Select:"+chr(9)+"YES"+chr(9)+"NO-attempted"+chr(9)+"NO-not attempted", client_contact_dropdown 'TO DO question do we want more HERE'
+  EditBox 70, 175, 145, 15, worker_signature
   ButtonGroup ButtonPressed
-    OkButton 110, 65, 45, 15
-    CancelButton 160, 65, 45, 15
-  Text 5, 30, 85, 10, "Mail Has Been Returned:"
-  Text 5, 50, 60, 10, "Worker Signature:"
+    OkButton 115, 195, 50, 15
+    CancelButton 165, 195, 50, 15
+    PushButton 135, 5, 80, 15, "SNAP TE02.08.012", SNAP_POLI_TEMP_button
+    PushButton 135, 25, 80, 15, "CASH TE02.08.011", CASH_POLI_TEMP_button
+    PushButton 135, 45, 80, 15, "ONEsource", one_source_button
   Text 5, 10, 50, 10, "Case Number:"
-  Text 100, 10, 50, 10, "Date Received:"
+  Text 5, 30, 50, 10, "Date Received:"
+  Text 5, 70, 85, 10, "Mail Has Been Returned:"
+  GroupBox 5, 85, 210, 85, " Request for Contact"
+  Text 10, 100, 140, 10, "Please make an attempt to call the client."
+  Text 10, 115, 200, 10, "SNAP: shelter cost must be verified to allow shelter deduction"
+  Text 10, 125, 195, 10, "CASH: address must be verified"
```

Comment:
I cannot find this in policy.

Reply:
MAXIS Cases
Returned mail with forwarding address
• 1. Call the household to confirm the current address information.
• 2. Update the case with the new address and forward the mail.
	• A. If the enrollee has moved out of your agency, follow current procedures to transfer the case to the new county or tribal agency.
	• B. If the enrollee has moved out-of-state, follow current procedures to close the case.
Returned mail with no forwarding address
• 1. If the mail was returned as undeliverable, compare the address on the returned mail with the following systems for the most up-to-date address:
	• • MAXIS
	• • METS (evidence dashboard, person record(s), case notes)
	• • Case file documents
	• • MMIS
• 2. If the address for the case contains errors or there is a more current address:
	• A. Update the case with the correct or new address and forward the mail.
• 3. If after following the steps above, there is no explanation why the mail was returned, call the household to confirm the current address information:
	• A. If new address information is provided, update the case with the new address, case note and forward the mail.
	• B. If they have moved out of your agency, follow the current procedure to transfer case to the new county or tribal agency.
	• C. If unable to make contact, follow steps in POLI/TEMP TE09.24.04 Whereabouts Unknown.
METS Cases
Returned mail with forwarding address
• 1. Call the household to confirm the current address information.
• 2. If the case is in IA/UQHP Blackout in Progress status, case note the new address and resend the returned mail to the correct address.
• 3. If the case is in Renewal in Progress status, [edit](http://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-17021402) the case with the new address and forward the mail.
	• A. If they have moved out of your agency, follow the [Transfer a Case](http://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-17021211) procedure in ONEsource.
	• B. If they moved out-of-state, follow the [Close Case – New Residential Address (Out of State)](http://www.dhs.state.mn.us/main/idcplg?IdcService=GET_DYNAMIC_CONVERSION&RevisionSelectionMethod=LatestReleased&dDocName=ONESOURCE-080306) procedure in ONEsource.